swarm/pss/pss_test.go

```diff
@@ -68,6 +71,75 @@ func init() {
 	wapi = whisper.NewPublicWhisperAPI(w)
 }
 
+func TestCache(t *testing.T) {
+	var err error
+	var potaddr pot.Address
+	to, _ := hex.DecodeString("08090a0b0c0d0e0f1011121314150001020304050607161718191a1b1c1d1e1f")
+	keys, err := wapi.NewKeyPair()
+	privkey, err := w.GetPrivateKey(keys)
+
+	ps := NewTestPss(privkey, nil)
+	pp := NewPssParams(privkey)
+	data := []byte("foo")
+	datatwo := []byte("bar")
+	fwdaddr := network.RandomAddr()
```

Comment:
I dont think there is any need for pot and pot.Address
just use the byteslice or `fwdadd.Over()`